### DISCUSSION TITLE
More Bungee compatability: specifically ignore, tablist, and a couple other minor things

### DISCUSSION THREAD

**Discussion by ghost** [Zrips/CMI#3482] [Feature Requests]
**Implementation of more Bungee features with CMI**

The main desire here is Ignore and Tablist, the others are just bonus awesome additions to see

---
### Detailed explanations ###
Basically the entire purpose is to turn some useful single-server features into bungee-server features. The **extreme** desire here is to get cross server /ignore. Currently ignoring only works if both players are on the same server, not different ones even if bungee chat is 100% handled by CMI. Other things would be bonuses and to die for, such as a tab list that would be global (can easily be hooked into an update to CMIB as well) so that tab lists can see players from all servers. Chat rooms handled by CMI are a sweet feature however they only work locally on their own server. This is fine and nice for sure, but would be a sweet feature to work cross server as well. 

It would also be nice to be able to target players from other servers as recipients to certain commands. I know we can tab complete them through the config, however it would be nice if that meant we also could use them as targets (basically CMI would search bungee network servers for that player as well if the player couldn't be found on the main server the command was run from). Such examples would be: heal, feed, give, god, fly, nick, pos, etc

### Basic explanation as to why this feature is needed and/or useful to have: ###
Ignore and tablist are most desired one for sake of moderation. I would gladly get behind the addition of this feature monetarily if needed as it is tremendously a great need of an add. Tablist is right up there with it as it will help with transparency of who's on what server since we can already tabcomplete and global chat, would be nice to customize a tablist too to see the other server. This would help cut down the other plugins 

The other targeting feature would just be super useful for sake of allowing players and staff to target players as recipients of CMI commands across the whole network. CMI is a GREAT plugin and I would overall just love to see more bungee aspects to it as well

**Comment by ablac** [comment]
I made a similar feature request, but I’d like to see a lot more implemented, like Bungee global Broadcast, Bungee global Toast, Bungee global bossbar etc.

I would also like to see the ability to teleport to players across the bungee network, as well as /MSG across bungee (It works sometimes but most the time it doesn’t) as well as a bunch of other commands, that would just make sense to have work as a unified server instead of multiple servers.

**Comment by EgidioBezerra** [comment]
Sync /kits over bungee too, so that players can't pickup the same /kit twice!
